src/heartbeat/utils.js

```diff
@@ -62,7 +58,20 @@ function changeCollectorStatus(currentStatus, newStatus) {
  */
 function updateCollectorConfig(cc) {
   const config = configModule.getConfig();
-  Object.keys(cc).forEach((key) => config.refocus[key] = cc[key]);
+  Object.keys(cc).forEach((key) => {
+    const oldValue = config.refocus[key];
+    const newValue = cc[key];
+    config.refocus[key] = cc[key];
+    if (oldValue && newValue !== oldValue) {
```

Comment:
Any special reason of oldValue && newValue... isn't be the same as newValue != oldValue?